bots/report-bundle-size.js

```diff
@@ -162,6 +167,60 @@ function android_getApkSize(engine, arch) {
   );
 }
 
+/**
+ * Returns whether the specified ref points to a pull request.
+ */
+function isPullRequest(ref) {
+  return ref !== 'main' && !/^\d+\.\d+-stable$/.test(ref);
+}
+
+/**
+ * Validates that required environment variables are set.
+ * @returns `true` if everything is in order; `false` otherwise.
+ */
+function validateEnvironment() {
```

Comment:
Wow! Thanks for really making this better! One thing is that posts-artifact-link script also uses this function.  https://github.com/facebook/react-native/blob/main/bots/post-artifacts-link.js#L25

We may want to even extract this function out and re-use it there.. and also update the `createOrUpdateComment` callsite